idea/resources/intentionDescriptions/ReplaceMathMaxWithCoerceAtLeastIntention/description.html

```diff
@@ -0,0 +1,5 @@
+<html>
+<body>
+    This intention detects 'Math.max' calls that can be safely replaced with 'coerceAtLeast'
```

Comment:
This intention replaces 'Math.max' calls with safe 'coerceAtLeast' ... or something like. Inspection detects, but inspection does some change.